### DISCUSSION TITLE
Unable to install jaeger-operator

### DISCUSSION THREAD

**Discussion by sharathbana** [jaegertracing/jaeger-operator#1668] [General]
I am trying to install jaeger-operator by following instruction on "https://www.jaegertracing.io/docs/1.29/operator/". But i get page not found error while doing so. Can someone please help me in solving this. As far as i know i am assuming that this config is not available in said location(git repo)

error: unable to read URL "https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/crds/jaegertracing.io_jaegers_crd.yaml", server reported 404 Not Found, status code=404

**Comment by frzifus** [comment]
could you try?
```bash
kubectl apply -f https://github.com/jaegertracing/jaeger-operator/releases/download/v1.29.0/jaeger-operator.yaml -n observability
```
Or if you want to download the file use:
```bash
wget https://github.com/jaegertracing/jaeger-operator/releases/download/v1.29.0/jaeger-operator.yaml
```
